osc-server/src/main/java/org/osc/core/broker/util/db/upgrade/ReleaseUpgradeMgr.java

```diff
@@ -517,17 +517,22 @@ private static void upgrade80to81(Statement stmt) throws SQLException {
     }
 
     private static void upgrade79to80(Statement stmt) throws SQLException {
-        execSql(stmt, "alter table DISTRIBUTED_APPLIANCE_INSTANCE DROP COLUMN IF EXISTS nsx_agent_id, "
-                + "nsx_host_id, nsx_host_name, nsx_host_vsm_uuid, nsx_vm_id;");
+        execSql(stmt, "alter table DISTRIBUTED_APPLIANCE_INSTANCE DROP COLUMN IF EXISTS nsx_agent_id;"
```

Comment:
alter table drop column should work with multiple columns  is it because of the if exists clause its not working?

http://www.h2database.com/html/grammar.html#alter_table_drop_column

Also we should follow the same pattern and run each sql statement under execSql

Reply:
Yes. Verified in SQL bench.

Reply:
without the if exists clause does it work?